packages/core/src/endowmentsToolkit.js

```diff
@@ -263,20 +329,29 @@ function endowmentsToolkit({
     return newPropDesc
   }
 
+  /**
+   *
+   * @param {PropertyDescriptor} sourcePropDesc
+   * @param {object} unwrapFromGlobalThis
+   * @param {object} unwrapToGlobalThis
+   * @returns {PropertyDescriptor}
+   */
   function applyGetSetPropDescTransforms(
     sourcePropDesc,
     unwrapFromGlobalThis,
     unwrapToGlobalThis
   ) {
     const wrappedPropDesc = { ...sourcePropDesc }
     if (sourcePropDesc.get) {
+      const getter = sourcePropDesc.get
       wrappedPropDesc.get = function () {
+        // eslint-disable-next-line @typescript-eslint/no-this-alias
         const receiver = this
         // replace the "receiver" value if it points to fake parent
         const receiverRef =
           receiver === unwrapFromGlobalThis ? unwrapToGlobalThis : receiver
         // sometimes getters replace themselves with static properties, as seen wih the FireFox runtime
-        const result = Reflect.apply(sourcePropDesc.get, receiverRef, [])
+        const result = Reflect.apply(getter, receiverRef, [])
```

Comment:
(must) Let's make sure this is not breaking the code. I'm pretty sure the comment above is explaining why we're not using a local copy but referencing the getter from the original source and this change would break it.

Reply:
Sorry this wasn't explained; TS' type inference doesn't know that `sourcePropDesc.get` is not `undefined` at this point, because it doesn't know when `wrappedPropDesc.get()` is called, so it cannot be sure, despite the conditional on L345 (or L346, depending).

I've changed it to just use a type assertion instead.

Reply:
By leaving it as-is, we're saying: _`sourcePropDesc.get` is allowed to be `undefined` here, and if so, allow `Reflect.apply()` to throw an exception_.  Just want to confirm that's the intent.

Reply:
We could doublecheck before applying. The breaking change was that we cached the getter instead of using the fresh one that could have been updated.